lumicks/pylake/tests/test_imaging_camera/test_correlated_stack.py

```diff
@@ -94,24 +135,33 @@ def compare_frames(original_frames, new_stack):
     compare_frames([0, 1, 2, 3, 4, 5], stack0[:-4])  # until negative index
     compare_frames([5, 6, 7], stack0[5:-2])  # mixed sign indices
     compare_frames([6, 7], stack0[-4:-2])  # negative indices slice
+    compare_frames([0, 2, 4, 6, 8], stack0[::2])  # all frames with steps
+    compare_frames([0, 3, 6, 9], stack0[::3])  # all frames with steps
+    compare_frames([3, 5], stack0[3:6:2])  # normal slice with steps
+    compare_frames([0, 2], stack0[:3:2])  # from beginning with steps
 
```

Comment:
Might be nice to add one that does recursive slicing w/ steps.

```suggestion
    compare_frames([0, 2], stack0[:3:2])  # from beginning with steps
    compare_frames([1, 5, 9], stack0[1::2][::2])  # recursive slice with steps
```